Resources/translations/LexikFormFilterBundle.cs.yml

```diff
@@ -0,0 +1,17 @@
+boolean:
+    yes_or_no:          "Ano nebo ne"
+    yes:                "Ano"
+    no:                 "Ne"
+
+number:
+    equal:              "Totožné"
+    greater_than:       "Větší než"
+    greater_than_equal: "Větší nebo rovno"
+    lower_than:         "Menší než"
+    lower_than_equal:   "Menší nebo rovno"
+
+text:
+    starts:             "Začíná na"
+    ends:               "Končí na"
+    equals:             "Schoduje se"
```

Comment:
Wouldn't `Rovná se` be correct here?

Reply:
I misspelled. It should be "Shoduje se".
It could be "Rovná se", "Rovno", "Stejný", "Totožný" or "Identický". There are a lots of [synonyms](http://www.slovnik-synonym.cz/web.php/slovo/shodny).
Maybe "Rovno" would be better in this case. It's hard to say.

Reply:
I've replaced it with "Rovno". Google translates "rovno" as "straight", but it is caused by missing context. It actually means straight but not in context of this form choice. If you put `"něco" rovno "něco"` Google translates it correctly as `"something" equal "something"`.